src/tagmanager/tm_parser.c

```diff
@@ -623,15 +646,39 @@ static TMParserMapEntry map_HTML[] = {
 	{'J', tm_tag_undef_t},      // script
 };
 static TMParserMapGroup group_HTML[] = {
-	{N_("Functions"), TM_ICON_NONE, tm_tag_function_t},  // javascript functions from subparser
 	{N_("Anchors"), TM_ICON_NONE, tm_tag_member_t},
 	{N_("H1 Headings"), TM_ICON_NONE, tm_tag_namespace_t},
 	{N_("H2 Headings"), TM_ICON_NONE, tm_tag_class_t},
 	{N_("H3 Headings"), TM_ICON_NONE, tm_tag_variable_t},
+	/* Embedded languages (JavaScript & PHP) */
+	{N_("Namespaces"), TM_ICON_NAMESPACE, tm_tag_package_t},
+	{N_("Interfaces"), TM_ICON_STRUCT, tm_tag_interface_t},
+	{N_("Classes"), TM_ICON_CLASS, tm_tag_typedef_t},
+	{N_("Functions"), TM_ICON_METHOD, tm_tag_function_t},
+	{N_("Constants"), TM_ICON_MACRO, tm_tag_macro_t},
+	{N_("Variables"), TM_ICON_VAR, tm_tag_union_t | tm_tag_local_var_t},
+	{N_("Traits"), TM_ICON_STRUCT, tm_tag_struct_t},
+	{N_("Properties"), TM_ICON_MEMBER, tm_tag_field_t},
```

Comment:
As the order here is the order of the groups in the tree, this actually changes where JS functions land.  I see 3 options here:

* have HTML stuff first, as it's the primary language here (e.g. what's in this diff)
* have functions on top as before, rest at the bottom
* have HTML at the bottom, everything else on top.

We actually could also try and split JS and PHP here, but that's gonna use a lot of `TMTagType`s, and might not add much.  And the order would still be a question :)

Opinions @techee @eht16 ?